Tests/GraphIR/example.gir

```diff
@@ -0,0 +1,29 @@
+module example where
+
```

Comment:
100% meant to be a slavish copy of SIL (but with `apply` as a terminator node!).  Textual GIR is intended to be implicitly scheduled - just like textual Thorin.  A graph representation has no canonical or stable textual form which makes testing nigh impossible. Instead, emission of GraphIR must be sent through a scheduler which emits this half-SSA-half-CPS text.  The overlying graph structure remains Thorin, but we are now free to parse this and either schedule it again - say after doing optimization passes - or accept pre-scheduled IR and run lowering passes on it to test LLVM IR generation.